releases/v19.2.0-beta.20190826.md

```diff
@@ -84,6 +84,13 @@ $ docker pull cockroachdb/cockroach-unstable:v19.2.0-beta.20190826
 - CockroachDB now supports an optional frame exclusion clause in the specification of window frames. [#28262][#28262] {% comment %}doc{% endcomment %}
 - Enabled collection of histograms by default during automatic statistics collection. [#39743][#39743]
 - The `force_retry()` function, which previously required admin privileges, is now available to all users to test their transaction retry loops. [#39246][#39246] {% comment %}doc{% endcomment %}
+- When [`DROP CONSTRAINT`](../v19.2/drop-constraint) is executed in a [transaction](../v19.2/transaction.html) on a validated constraint, it no longer takes effect immediately, and subsequent writes in that constraint will now fail if they violate that constraint. This is to enforce consistency when dropping constraints asynchronously in the schema changer. [#39648][#39648]
+- The `ALTER PARTITION` [statement](../v19.2/sql-statements.html) now supports applying a [zone configuration](../v19.2/configure-replication-zones.html) to all the partitions of a table and its indexes that share the same partition name. The syntax for this is `ALTER PARTITION <partition name> OF INDEX <table name>@*`. [#39750][#39750]
+- Support parsing the `FOR UPDATE` modifier on [`SELECT`](../v19.2/select-clause.html) clauses, treating it as a no-op, since CockroachDB's [transactions](../v19.2/transactions.html) only operate in `SERIALIZABLE` mode. [#40206][#40206]
```

Comment:
Worked with Ben to flesh this out a bit. Please add links back in:

> CockroachDB now supports the `FOR UPDATE` modifier on `SELECT` statements for compatibility with third-party tools. It has no effect because `SERIALIZABLE` transactions do not require explicit write locks to maintain consistency. However, this means that `SELECT FOR UPDATE` cannot be used as an advisory exclusive lock in CockroachDB, as used by some tools.